internal/validations/validations.go

```diff
@@ -288,6 +290,23 @@ func validateNotStatic(_ context.Context, _ string, baton *Baton) *types.Validat
 	return types.NewValidationError(types.ErrNotDynLinked)
 }
 
+func validateExcludedCryptoModules(ctx context.Context, path string, _ *Baton) *types.ValidationError {
+	var stdout bytes.Buffer
+	cmd := exec.CommandContext(ctx, "nm", "-j", path)
```

Comment:
It was found in the initial scanner implementation that object level symbols are not reliable enough, since they can be stripped from the binary.  Because of this, function data is read by the scanner from the Go runtime's PCLN table instead.  See [validateGoSymbols](https://github.com/openshift/check-payload/blob/d67cb51bcbfb5c115582f6bf46d8c65c6c14b3fe/internal/validations/validations.go#L76-L99).

In the case of the current validation, something like this could probably do the trick:
```
var symbols []string
excluded := "golang.org/x/crypto"
symtable, err := golang.ReadTable(path, baton.GoBuildInfo)
if err != nil {
	return types.NewValidationError(fmt.Errorf("go: could not read table for %v: %w", filepath.Base(path), err))
}
for _, f := range symtable.Funcs {
	if strings.Contains(f.Name, excluded) {
		return types.NewValidationError(types.ErrDetectedExcludedModule)
	}
}

```